pkg/operator/api/api.go

```diff
@@ -161,3 +161,7 @@ func getLabels(clusterName string) map[string]string {
 		k8sutil.ClusterAttr: clusterName,
 	}
 }
+
+func MakeRookOperatorImage(version string) string {
```

Comment:
could you put this in the k8sutil package? Then we'll have both these methods for rookd and rook-operator next to each other